mozilla/blurts-server
# Add unique page titles

# References: 
Jira: MNTOR-1486
Figma: N/A




# Description

Adds unique page titles and social media descriptions to every page. I still have an open question with Mela that Florian is CC'd on, and Florian will take on getting this PR over the finish line since I'll be out.

# How to test

Visit the different page, and check that their metadata and tab titles are unique.

# Checklist (Definition of Done)
